GetPlayersInSegmentSample/get_players_in_segment_api_sample.js

```diff
@@ -0,0 +1,59 @@
+handlers.GetPlayersInSegmentSample = function (args, context) {
+
+    /*
+        Sample code to use the GetPlayersInSegment API to the process player profiles in a Segment.
+        The GetPlayersInSegment API pages through the all the player profiles
+        in the Segment in batches of size 'MaxBatchSize'.
+        API Doc: https://learn.microsoft.com/en-us/rest/api/playfab/server/play-stream/get-players-in-segment?view=playfab-rest
+    */
+
+    var request =  {
+     MaxBatchSize: 1000,
+     SegmentId: "AAAAAAAAA" // provide your SegmentId here
+    }
+
+    // make the first GetPlayersInSegment API call
+    var playersInSegmentResult =  server.GetPlayersInSegment(request);
+    
+    // process until continuation token is not null to get all the profiles in this Segment
+    while (playersInSegmentResult.ContinuationToken != null) 
+    {
+        // get the current batch of player profiles
+        var playerProfiles = playersInSegmentResult.PlayerProfiles;
+
+        // TODO: 
+        // Add logic here to process the above playerProfiles
```

Comment:
This is a TODO for the developer, right? (not for you...)

Do you want to add a little bit of code to show handling the short-circuit behavior? would playerProfiles be null or an empty array?

if (playerProfiles != null) {
  // process player profiles here
}

Reply:
Added the empty check